src/main/tray.ts

```diff
@@ -78,28 +75,6 @@ function createTrayMenu(windowManager: WindowManager): Menu {
           .catch(error => logger.error(`${TRAY_LOG_PREFIX}: Failed to nativate to Preferences`, { error }));
       },
     },
-    {
```

Comment:
Why do we want to remove this?

Reply:
Because we don't have workspaces/clusters anymore... we can add replacement if/when we know what should be here.

Reply:
Oh okay